SPECS/libarrow/libarrow.spec

```diff
@@ -0,0 +1,366 @@
+%define _unpackaged_files_terminate_build 0 
+%bcond_without use_flight
+%bcond_with use_plasma
+%bcond_with use_gandiva
+%bcond_with use_mimalloc
+%bcond_without use_ninja
+# TODO: Enable this. This works on local but is fragile on GitHub Actions and
+# Travis CI.
+%bcond_with use_s3
+%bcond_without have_rapidjson
+%bcond_without have_re2
+%bcond_without have_utf8proc
+ 
+Name:	libarrow
+Version:	15.0.0
+Release:	4%{?dist}
+Summary:	A toolbox for accelerated data interchange and in-memory processing
+License:	Apache-2.0
```

Comment:
Can you please add the following to match our other specs:
Group:
Vendor:
Distribution:

Reply:
added them